crates/ef-testing/src/models/case.rs

```diff
@@ -92,16 +95,27 @@ impl BlockchainTestCase {
         tx_signed.signature = signature;
 
         let execution_result = sequencer.execute_transaction(tx_signed);
-        log_execution_result(execution_result, &self.case_name, &self.case_category);
+        log_execution_result(&execution_result, &self.case_name, &self.case_category);
 
-        Ok(())
+        let retdata = extract_execution_retdata(execution_result);
+
+        Ok(retdata)
     }
 
-    fn handle_post_state(&self, sequencer: &mut KakarotSequencer) -> Result<(), RunnerError> {
+    fn handle_post_state(
+        &self,
+        sequencer: &mut KakarotSequencer,
+        retdata: Option<String>,
+    ) -> Result<(), RunnerError> {
         let wallet = LocalWallet::from_bytes(&self.secret_key.0)
             .map_err(|err| RunnerError::Other(vec![err.to_string()].into()))?;
         let sender_address = wallet.address().to_fixed_bytes();
 
+        let eth_validation_failed = retdata
+            .clone()
```

Comment:
you can fix clippy but removing the clone

Reply:
I ran clippy locally but didn't have any warnings, are there specific params?